docs/deprecations.md

```diff
@@ -26,7 +26,7 @@ being deprecated.
 | [`v1alpha1.Run` is deprecated, feature flag `enable-custom-task` is removed in v0.44.0, `custom-task-version` will be removed together with `v1alpha1.Run`](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md) | v0.43.0                                                              | Alpha                                                                                                 | April 10, 2023 or v0.47.0           |
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
-
+| [`config-trusted-resources` is deprecated](https://github.com/tektoncd/pipeline/issues/5852)                                                                                                                 | v0.45.0                                                              | Alpha                                                                                                  | April 10, 2023 or v0.47.0                       |
```

Comment:
Do we need to update the alpha features table to explain that this feature is going to be removed? I'm also confused what the "config-trusted-resources" feature actually is. Is that the feature enabled by the flag "resource-verification-mode"? How would someone use VerificationPolicy, if that's the replacement feature?
Lastly, why are we deciding to support this for two more months?

Reply:
- This feature stores public keys in config map, it's a subset feature of VerificationPolicy. 
- I will update the docs as @JeromeJu suggested. 
- The announcement is 0.45, are we ok to remove it in 0.46? If so I think we could change it to 0.46

Reply:
sg, please lmk when you've updated the PR to make it more clear how to use the replacement feature. re: when can we remove it? we can remove alpha features at any time according to our compatibility policy, so I'd put N/A (alpha)